### ISSUE TITLE
find_project_root may cause error

### ISSUE THREAD

**Issue by ericmtsai** [sfsinger19103/election_anomaly#45]
`find_project_root()` takes a guess at the project root by splitting on 'election_anomaly' in the path. For example, if a directory is named 'my_election_anomaly_finder', there may be issues.

**Comment by ericmtsai** [comment]
Not sure that this is still an issue in the new feature branch. I did a grep for this function and couldn't find it.

**Comment by sfsinger19103** [comment]
function is `get_project_root`. It's still there.

**Comment by ericmtsai** [comment]
Yes, I see it now. It looks like this is never called now since we're using the new approach - the project root parameter is required and read directly from the run_time.par file. I changed the title to tech debt to reflect this. We can keep this issue here as a reminder for cleanup purposes. But I will remove the Beta Milestone and consider this lower priority unless I hear otherwise.

**Comment by sfsinger19103** [comment]
get_project_root() and the various places it is used in code (`if project_root is None: project_root=get_project_root()`) are now obsolete. If project root is not given in `run_time.par`, failure is graceful. 

```
>>> d = e.DataLoader(); d.check_errors()
Parameter file missing requirements.
{'missing': ['project_root']}
DataLoader object not created.
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'check_errors'
>>> 
```
